packages/cli/src/utils.ts

```diff
@@ -252,18 +252,11 @@ const getAndroidDetails = async (
   const minSdk = new RegExp(
     AaptPrefixes.sdkPrefix + AaptPrefixes.quoteRegex
   ).exec(stdout);
-  const permissions = new RegExp(
-    AaptPrefixes.permissionPrefix + AaptPrefixes.quoteNonLazyRegex
-  ).exec(stdout);
+  const permissions = [...stdout.matchAll(/uses-permission: name='(.*)'/g)];
```

Comment:
We still have `AaptPrefixes.permissionPrefix` defined above. Recommend either continuing to use it to assemble the string, or remove it entirely.

Reply:
Removed the string.